research-software-directory/RSD-as-a-service
# feat: admins can set parent organisation

# Admins can set parent organisation in frontend

Closes #1177 

Changes proposed in this pull request:

* adds a feature for administrators to be able to set the parent organisation of an organisation
* visually separates the block with administrator settings under "General settings" 

![image](https://github.com/research-software-directory/RSD-as-a-service/assets/14222414/74762c2d-1bf7-4bef-9f93-55165ea4d4f5)

How to test:

* `docker compose build --parallel && docker compose up --scale scrapers=0`
* `docker compose run data-generation`
* login as administrator
* open an organisation page and navigate to settings
* copy the organisation id from within the admin panel\
* open another organisation's settings and paste the organisation id into the "Parent organisation ID" field
* verify that the organisation is now a child of the other organisation
* alter the organisation id so that it is invalid => error is shown
* remove the parent organisation id and verify that the organisation is now not a child organisation anymore
* check the corresponding sections in the documentation (http://localhost:3000//documentation/rsd-instance/administration/#edit-organisation and http://localhost:3000/documentation/users/organisation/#what-can-organisation-maintainers-do)

PR Checklist:
